config/manager/manager.yaml

```diff
@@ -3,6 +3,9 @@ kind: Namespace
 metadata:
   labels:
     control-plane: controller-manager
+    openshift.io/scc: "anyuid"
```

Comment:
I'd like to see a comment describing why we're setting these values with a link to the documentation. Do you think that would be too invasive?